README.md

```diff
@@ -128,74 +177,39 @@ Generate a `Red Hat Dependency Analytics Report` for your application by perform
+	<br >The Red Hat Dependency Analytics report is a temporary HTML file that exist if the **Red Hat Dependency Analytics Report** tab remains open.
+	Closing the tab removes the temporary HTML file.
+	You can specify the file name by [modifying the _Red Hat Dependency Analytics: Red Hat Dependency Analytics Report File Path_ field](#configuration) in the extension settings.
+	The default location for this file is `/tmp/redhatDependencyAnalyticsReport.html`.
 
-## Configuration
-
-The `Red Hat Dependency Analytics` plugin has configurable parameters within the extension that allow users to tailor the behavior and functionality of the extension according to their preferences.
-To access these configurable parameters please enter the [extension settings](https://code.visualstudio.com/docs/getstarted/settings) within your VS code instance, switch to *Workspace* tab and search for *Red Hat Dependency Analytics*.
-
-![Red Hat Dependency Analytics Extension Workspace Settings](images/screenshots/extension-workspace-settings.png)
-
-### Configurable Parameters
-
-**Exhort Snyk Token** - edit the *Exhort Snyk Token* setting to change the Snyk token setting.
-The Snyk Token allows Exhort to authenticate with Snyk (vulnerability data provider).
-Please note that if a valid Snyk Token is not provided in the extension workspace settings, Snyk vulnerabilities will not be displayed.
-An alert message on edit will provide feedback on whether the token is valid or not.
-To generate a new token please visit the following [link](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
-
-**Valid Token Info Message**<br >
-![ screenshot ](images/screenshots/valid-token.png)
-
-**Invalid Token Alert Message**<br >
-![ screenshot ](images/screenshots/invalid-token.png)
-
-**Enpty Token Warning Message**<br >
-![ screenshot ](images/screenshots/empty-token.png)
+## Using Red Hat Dependency Analytics for CI builds
 
-**Red Hat Dependency Analytics Report File Path** - edit the *Red Hat Dependency Analytics Report File Path* setting to change the location where the HTML of the `Red Hat Dependency Analytics Report` will be stored (Default value: `/tmp/redhatDependencyAnalyticsReport.html`).
-
-## Using Red Hat Dependency Analytics on your CI Builds
-
-To automate the analysis of vulnerabilities in a project's dependencies, we offer integration with the following Continuous Integration (CI) platforms:
+You can automate the analysis of your application's vulnerabilities within the build and release pipeline.
+Red Hat offers integration with these Continuous Integration (CI) platforms:
 
 - [Red hat Dependency Analytics Tekton Task](https://hub.tekton.dev/tekton/task/redhat-dependency-analytics)
 
 - [Red hat Dependency Analytics Jenkins Plugin](https://plugins.jenkins.io/redhat-dependency-analytics/)
 
-- Red hat Dependency Analytics GitHub Actions \<TBD\>
```

Comment:
Okay, I'll add it back, slightly reworded.  Typically, we don't mention TBD items in technical documentation, because it's a vague and ambiguous statement, that's leaves the user scratching their head and asking "why".  It's better not to mention it at all.  This sounds like a unique case, so I understand if this was an available feature, then taken away, it's important to mention that.

Reply:
I added an important admonishment explaining that GitHub actions are temporarily disabled.